plugin/storage/badger/options.go

```diff
@@ -19,18 +20,33 @@ type Options struct {
 	// This storage plugin does not support additional namespaces
 }
 
-// NamespaceConfig is badger's internal configuration data
+// NamespaceConfig is badger's internal configuration data.
 type NamespaceConfig struct {
-	namespace      string
-	SpanStoreTTL   time.Duration `mapstructure:"span_store_ttl"`
-	ValueDirectory string        `mapstructure:"directory_value"`
-	KeyDirectory   string        `mapstructure:"directory_key"`
-	// Setting this to true will ignore ValueDirectory and KeyDirectory
-	Ephemeral             bool          `mapstructure:"ephemeral"`
-	SyncWrites            bool          `mapstructure:"consistency"`
-	MaintenanceInterval   time.Duration `mapstructure:"maintenance_interval"`
+	namespace string
```

Comment:
do we want to make this field configurable?

Reply:
it's not needed, see https://github.com/jaegertracing/jaeger/pull/5929